### PR TITLE
change method of opening side content 

### PR DESCRIPTION
**Pull request by TurnbulA**

### CODE REVIEW

File: app/js/toDoToggle.js [hunk 1]
```diff
@@ -11,9 +11,9 @@ const openProject = () => {
         }
       });
       renderToDoList(projectTarget);
-      const projectToDo = document.querySelector(".c-project__list");
+      const projectToDo = document.querySelector(".c-project__container--list");
       projectIconBody.classList.add("c-project__container--left-justify");
-      projectToDo.classList.add("c-project__list--active");
+      projectToDo.classList.add("c-project__container--list--left-justify");
```
Comment (mali891) [inline app/js/toDoToggle.js]: This is...this is two modifiers

File: index.html [hunk 2]
```diff
@@ -58,152 +58,155 @@
...
 
     <div class="c-container">
-      <div class="c-project__body">
-        <div class="c-header">
-          <h1 class="c-header--title">Welcome Back</h1>
-          <p class="c-header--para">We Have Missed You</p>
-          <form class="c-search-bar">
-            <button type="submit" class="c-search-bar__button">
+      <div class="c-project__container">
+        <div class="c-project__container--body">
```
Comment (mali891) [inline index.html]: `--body` isn't a modifier

File: index.html [hunk 3]
```diff
@@ -58,152 +58,155 @@
...
+          <div class="c-header">
+            <h1 class="c-header--title">Welcome Back</h1>
```
Comment (mali891) [inline index.html]: `--title` isnt a modifier

File: index.html [hunk 4]
```diff
@@ -58,152 +58,155 @@
...
+            <p class="c-header--para">We Have Missed You</p>
```
Comment (mali891) [inline index.html]: `--para` isnt a modifier

File: index.html [hunk 5]
```diff
@@ -58,152 +58,155 @@
...
+                </svg>
+              </button>
+              <input
+                class="c-search-bar__input"
+                type="text"
+                placeholder="Search Task or Project..."
+              />
+            </form>
+            <div class="c-search-bar__container">
+              <p class="c-search-bar__container--text"></p>
```
Comment (mali891) [inline index.html]: `--text` isnt a modifier

File: index.html [hunk 6]
```diff
@@ -58,152 +58,155 @@
...
+              <button class="c-search-bar__container--reset-button">
```
Comment (mali891) [inline index.html]: `--reset-button` isnt a modifier

File: index.html [hunk 7]
```diff
@@ -58,152 +58,155 @@
...
-                  fill="#9769c7"
-                />
-                <g transform="translate(-70 -6.5)">
-                  <line
-                    x2="11"
-                    transform="translate(1800 849)"
-                    fill="none"
-                    stroke="#fff"
-                    stroke-width="1.5"
+        <div class="c-project__container--list">
```
Comment (mali891) [inline index.html]: `--list` isnt a modifier

File: index.html [hunk 8]
```diff
@@ -58,152 +58,155 @@
...
+          <div class="c-projects-to-do">
+            <div class="c-projects-to-do__header">
+              <div class="c-projects-to-do__header--text-content"></div>
```
Comment (mali891) [inline index.html]: `--text-content` isnt a modifier

File: index.html [hunk 9]
```diff
@@ -58,152 +58,155 @@
...
+              <div class="c-projects-to-do__header--close">
```
Comment (mali891) [inline index.html]: `--close` may or may not be a modifier - what does it do?

File: index.html [hunk 10]
```diff
@@ -58,152 +58,155 @@
...
               <svg
                 xmlns="http://www.w3.org/2000/svg"
-                width="30"
-                height="30"
-                viewBox="0 0 30 30"
-                class="c-projects-to-do__close--cross"
+                width="43"
+                height="44"
+                viewBox="0 0 43 44"
+                class="c-projects-to-do__list-item--svg"
```
Comment (mali891) [inline index.html]: `--svg` isnt a modifier

File: index.html (hunk 9 above)
Comment (TurnbulA) [reply index.html]: close svg

**Pull request closed without merging**